packages/react-bootstrap-table2/src/body.js

```diff
@@ -27,8 +28,12 @@ const Body = (props) => {
   } else {
     content = data.map((row, index) => {
       const key = _.get(row, keyField);
-      const editable = !(cellEdit && cellEdit.nonEditableRows.indexOf(key) > -1);
-      const selected = selectedRowKeys.includes(key);
+      const editable = !(cellEdit.mode !== Const.UNABLE_TO_CELL_EDIT &&
```

Comment:
Hi @AllenFang:

With [props-resolver](https://github.com/react-bootstrap-table/react-bootstrap-table2/blob/develop/packages/react-bootstrap-table2/src/props-resolver/index.js#L38-L41), `cellEdit` is always be an Object with different `mode`. It seems to be a little wired to judge cell-editable by `cellEdit`. Therefore, I make this modification. If you have any concerns or other suggestion plz let me know, thx 👍

Reply:
👍